test/e2e/framework/volume/fixtures.go

```diff
@@ -696,3 +697,71 @@ func GetLinuxLabel() *v1.SELinuxOptions {
 	return &v1.SELinuxOptions{
 		Level: "s0:c0,c1"}
 }
+
+// CheckVolumeModeOfPath check mode of volume
+func CheckVolumeModeOfPath(f *framework.Framework, pod *v1.Pod, volMode v1.PersistentVolumeMode, path string) {
+	if volMode == v1.PersistentVolumeBlock {
+		// Check if block exists
+		VerifyExecInPodSucceed(f, pod, fmt.Sprintf("test -b %s", path))
+
+		// Double check that it's not directory
+		VerifyExecInPodFail(f, pod, fmt.Sprintf("test -d %s", path), 1)
+	} else {
+		// Check if directory exists
+		VerifyExecInPodSucceed(f, pod, fmt.Sprintf("test -d %s", path))
+
+		// Double check that it's not block
+		VerifyExecInPodFail(f, pod, fmt.Sprintf("test -b %s", path), 1)
+	}
+}
+
+// TODO: put this under e2epod once https://github.com/kubernetes/kubernetes/issues/81245
+// is resolved. Otherwise there will be dependency issue.
```

Comment:
@chrishenzie Replying to your concern on this function:
There are two possible location I think this function could lie in other than this:
1. test/e2e/storage/utils/pod.go
The problem with that is this function is being used by "VerifyExecInPodSucceed" and "VerifyExecInPodFail" which are used by "CheckVolumeModeOfPath".

And this "CheckVolumeModeOfPath" is used by e2evolume. So ideally we dont want to put them under storage utils because that will have a situation where parent package import subpackage. This against our principle.

2. test/e2e/framework/pod
This is a better place for sure. However, currently there is a dependency issue #81245 which prevent us from putting it there. So I add a comment to this. Once that is fixed we can move it over.